### ISSUE TITLE
Publish thyme on maven

### ISSUE THREAD

**Issue by rklaehn** [Ichoran/thyme#12]
I love Thyme and use it extensively in my private and work projects. I would like to use it more in spire.

However, the current situation in spire is less than optimal. Thyme is downloaded from the private webserver of @non . 
```
      // thyme
      "ichi.bench" % "thyme" % "0.1.0" from "http://plastic-idolatry.com/jars/thyme-0.1.0.jar",
```
In my private and work projects I have an unmanaged test-lib project and check in thyme.jar into git. Which is also far from optimal.

The obvious solution would be to publish Thyme on maven.

**Comment by Ichoran** [comment]
Agreed.  I haven't had time to look in to that yet.  I'll try to get to it in July.

That said, Maven itself is less than optimal, and just having a URL to grab it from is not _so_ much worse.  Thus, I don't promise to get to it in July.

**Comment by RichardBradley** [comment]
In the meantime, you can add the GitHub hosted JAR to a SBT project with:

    "ichi.bench" % "thyme" % "0.1.1" from "https://github.com/Ichoran/thyme/raw/master/Thyme.jar"

**Comment by rklaehn** [comment]
Would you accept a PR to the build.sbt so you could push it to maven via sonatype (assuming you have credentials there)?

**Comment by Ichoran** [comment]
@rklaehn - I don't have credentials there, but they're not extremely hard to get.  So, yes.
